packages/fxa-admin-panel/server/lib/server.ts

```diff
@@ -71,13 +81,52 @@ app.get('/__version__', (_, res) =>
   res.type('application/json').send(JSON.stringify(version))
 );
 
+function injectMetaContent(html: any, metaContent: any = {}) {
+  let result = html;
+
+  Object.keys(metaContent).forEach(k => {
+    result = result.replace(
+      k,
+      encodeURIComponent(JSON.stringify(metaContent[k]))
+    );
+  });
+
+  return result;
+}
+
+function injectHtmlConfig(html: any, config: any) {
```

Comment:
No `any`s unless you truly don't know their types.

Reply:
Looking at the calls, I think you could just let TS to infer the types here and at `injectMetaContent`?